src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -22,8 +22,12 @@
 (defn winner? [board]
   (some? (some #(apply = %) (potential-wins board))))
 
+
+(defn check-for-equality [potential-wins board]
```

Comment:
what's considered "equality"?

Reply:
Hah, no idea what happened there or why that didn't break the tests when I ran them the first time. Fixed it to what it should have been.